litmus-portal/frontend/src/pages/WorkflowDetails/index.tsx

```diff
@@ -26,35 +26,32 @@ import useStyles from './styles';
 import NodeTable from '../../views/WorkflowDetails/workflowTable';
 import WorkflowInfo from '../../views/WorkflowDetails/WorkflowInfo';
 import NodeLogsModal from '../../views/WorkflowDetails/LogsModal';
-import * as ToggleButtonAction from '../../redux/actions/button';
 
 const WorkflowDetails: React.FC = () => {
   const theme = useTheme();
   const { t } = useTranslation();
   const classes = useStyles();
-  const [logsModalOpen, setLogsModalOpen] = useState(false);
+  const [logsModalOpen, setLogsModalOpen] = useState<boolean>(false);
+  const [isInfoToggled, setIsInfoToggled] = useState<boolean>(true);
+  const [workflowRunId, setWorkflowRunId] = useState<String>('');
 
   const tabs = useActions(TabActions);
-  const toggleButtonAction = useActions(ToggleButtonAction);
-
-  // Getting the workflow nome from the pathname
-  const { pathname } = useLocation();
-  const workflowRunId = pathname.split('/')[2];
-
   // get ProjectID
   const selectedProjectID = useSelector(
     (state: RootState) => state.userData.selectedProjectID
   );
 
-  // For showing details based on togglingInfo
-  const { isInfoToggled } = useSelector(
-    (state: RootState) => state.toggleInfoButton
-  );
-
   const workflowDetailsTabValue = useSelector(
     (state: RootState) => state.tabNumber.node
   );
 
+  // get workflowRunID from localforage
+  useEffect(() => {
+    localforage.getItem('selectedWorkflowRunID').then((value) => {
```

Comment:
why is selectedWorkflowRunID taken from localforage...shouldn't it be already available in the URL?